apps/api/src/services/data-provider/manual/manual.service.ts

```diff
@@ -166,27 +166,26 @@ export class ManualService implements DataProviderInterface {
         }
       });
 
-      const symbolProfilesWithScraperConfiguration = symbolProfiles.filter(
-        ({ scraperConfiguration }) => {
-          return !!scraperConfiguration?.url;
-        }
-      );
+      const symbolProfilesWithScraperConfigurationAndInstantMode =
+        symbolProfiles.filter(({ scraperConfiguration }) => {
+          return scraperConfiguration?.mode === 'instant';
```

Comment:
@shaun-ak Sorry for the confusion, we actually forgot to check the `mode` 😄

Reply:
Oh, yes 😊